js/controller.js

```diff
@@ -260,6 +268,15 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
+
+      if (this.videoVr) {
+        this.mb.subscribe(OO.EVENTS.DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
```

Comment:
OO.EVENTS.DIRECTION_CHANGED: I'm not sure where this event is defined, but can we rename it to include a mention of VR?

Reply:
This functionality together with your wishes is moved to feature / PLAYER-1422